### PR TITLE
Apply link formatting to Solid World April's links

### PR DESCRIPTION
**Pull request by Vinnl**

### CODE REVIEW

File: pages/solid-events.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Are you interested in speaking about what you have been building on Solid? Reach
...
+| 2021-04-01 | [Solid World April](https://www.eventbrite.com/e/solid-world-april-2021-tickets-144512880917)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/solidworld/review/531907097/eda903e37f)<br/>Mentioned links: Tim's talk: [Web's 32nd Birthday](https://webfoundation.org/2021/03/web-birthday-32/), Oz’s talk: [Survey for the Universal API](https://www.research.net/r/GWWJB7V); Graphmetrix' talk: [Website](https://graphmetrix.com/), [sign up for a Trin Pod](https://trinpod.us/); Nextcloud/PDS Interop's talk:[PDS Interop Website](https://pdsinterop.org), [PDS Interop](https://github.com/pdsinterop), [Yvo Brevoort](https://github.com/ylebre), [Solid Test Suite](https://github.com/solid/test-suite), [PHP Solid Server](https://github.com/pdsinterop/php-solid-server), [Nextcloud Solid Server](https://github.com/pdsinterop/solid-nextcloud), [slides](http://solid-world.dev.muze.nl/); Spoter.Me's talk: [homepage](https://spoter.me).|
```
Comment (kay-kim) [inline pages/solid-events.md:24]: This is out of scope but just idle wondering 

- is the trinpod.us link correct?  or should it be https://graphmetrix.com/trinpod ?
- also, is it odd to link Yvo's profile in the mentioned links page?  or maybe have it be [Yvo Brevoort's github profile] or something.

**Review by kay-kim — APPROVED** [review]
LGTM

File: pages/solid-events.md (hunk 1 above)
Comment (Vinnl) [reply pages/solid-events.md:24]: @MarrelleBailey I think you added those links, so probably in the best position to answer? :)

Comment (MarrelleBailey) [reply pages/solid-events.md]: Hi, back from vacation sorry for the late reply. @kay-kim thanks @Vinnl for pointing this out. 

1. We can use the graphmetrix.com/trinpod but I am using the link they shared with me which is https://trinpod.us/
2. He shared the profile but if it's not relevant we don't have to include it.

**Pull request merged**